clients/errors.go

```diff
@@ -0,0 +1,10 @@
+package clients
+
+import "errors"
+
+// TODO: add more errors that are duplicated in different clients
```

Comment:
maybe move them in a package like elrond-eth-bridge/errors ?

Reply:
I think it is better for this particular kind of errors to have them here. We can reuse those errors for any clients we may add in the future. 

*The errors that are particular for Elrond <-> Eth bridge shall definitely placed in a package as you suggested